create-doks-with-terraform-flux/README.md

```diff
@@ -1 +1,470 @@
+
+    Let's initialize the backend now using the previously created `access` and `secret` keys for the [DO Spaces](https://cloud.digitalocean.com/spaces):
+   
+    ```bash
+    terraform init  --backend-config="access_key=$DO_SPACES_ACCESS_KEY" --backend-config="secret_key=$DO_SPACES_SECRET_KEY"
+    ```
+
+    Sample output:
+    ```
+    Initializing the backend...
+
+    Successfully configured the backend "s3"! Terraform will automatically
+    use this backend unless the backend configuration changes.
+
+    Initializing provider plugins...
+    - Finding hashicorp/kubernetes versions matching "2.3.2"...
+    - Finding gavinbunney/kubectl versions matching "1.11.2"...
+    ...
+    ```
+6. Going furher let's create a `plan` in order to inspect the infrastructure changes:
+
+    ```bash
+    terraform plan
+    ```
+7. If everything seems alright then `apply` the changes with: 
+   
+    ```bash
+    terraform apply
+    ```
+
+    Sample output:
+    ```
+    tls_private_key.main: Creating...
+    kubernetes_namespace.flux_system: Creating...
+    github_repository.main: Creating...
+    tls_private_key.main: Creation complete after 2s [id=1d5ddec06b0f4daeea57d3a987029c1153ebcb21]
+    kubernetes_namespace.flux_system: Creation complete after 2s [id=flux-system]
+    kubectl_manifest.install["v1/serviceaccount/flux-system/source-controller"]: Creating...
+    kubectl_manifest.sync["kustomize.toolkit.fluxcd.io/v1beta1/kustomization/flux-system/flux-system"]: Creating...
+    kubectl_manifest.install["v1/serviceaccount/flux-system/helm-controller"]: Creating...
+    kubectl_manifest.install["networking.k8s.io/v1/networkpolicy/flux-system/allow-egress"]: Creating...
+    ...
+    ```
+
+# Next steps
+
+## Seeing the results
+If everything goes well the [DOKS cluster](https://cloud.digitalocean.com/kubernetes/clusters) should be up and running as well as [Flux CD](https://fluxcd.io):
+
+![DOKS state](content/img/doks_created.png)
+
+The terraform state file should be saved in your [DO Spaces](https://cloud.digitalocean.com/spaces) bucket so please go ahead and check it. It should look similar as seen in the picture down below:
+
+![DO Spaces Terraform state file](content/img/tf_state_s3.png)
+
+A new git repository should be created as well for you containing the Flux CD cluster resource files:
+
+![GIT repo state](content/img/flux_git_res.png)
+
+## Inspecting the cluster and applications state
+In order to inspect the Kubernetes cluster as well as the Flux CD state and getting information about various components we need to install a few tools like:
+
+1. `doctl` for DigitalOcean interaction (most of the tasks that can be done via the DO account web interface can be accomplished using the CLI version as well) 
+2. `kubectl` for Kubernetes interaction
+3. `flux` for Flux CD interaction
```

Comment:
```suggestion
3. `flux` for Flux interaction
```

Reply:
Thanks for the suggestion. Fixed in the latest commit.